### PR TITLE
Fixing typos, uniformisation of help text style, replace PR #2699

### PR DESCRIPTION
**Pull request by imifos**
Replaces https://github.com/ethereum/cpp-ethereum/pull/2699

### CODE REVIEW

File: eth/main.cpp [hunk 1]
```diff
@@ -106,74 +108,74 @@ void help()
...
 		<< "Client structured logging:" << endl
-		<< "    --structured-logging  Enable structured logging (default output to stdout)." << endl
+		<< "    --structured-logging  Enable structured logging (default: output to stdout)." << endl
 		<< "    --structured-logging-format <format>  Set the structured logging time format." << endl
 		<< "    --structured-logging-url <URL>  Set the structured logging destination (currently only file:// supported)." << endl
 		<< endl
 		<< "Import/export modes:" << endl
-		<< "    --from <n>  Export only from block n; n may be a decimal, a '0x' prefixed hash, or 'latest'." << endl
-		<< "    --to <n>  Export only to block n (inclusive); n may be a decimal, a '0x' prefixed hash, or 'latest'." << endl
+		<< "    --from <n>  Export only from block n; n may be a decimal, a '0x' pre-fixed hash, or 'latest'." << endl
```
Comment (gavofyork) [inline eth/main.cpp]: "prefixed" (no hyphen) is fine. see [definition of "to prefix" verb](http://www.thefreedictionary.com/prefix).

File: eth/main.cpp [hunk 2]
```diff
@@ -492,7 +494,7 @@ int main(int argc, char** argv)
 				cerr << "-c is DEPRECATED. It will be removed for the Frontier. Use --client-name instead." << endl;
 			clientName = argv[++i];
 		}
-		else if ((arg == "-a" || arg == "--address" || arg == "--coinbase-address") && i + 1 < argc)
+		else if ((arg == "-a" || arg == "--address" || arg == "--etherbase-address" || arg == "--coinbase-address") && i + 1 < argc)
```
Comment (gavofyork) [inline eth/main.cpp]: prefer `--beneficiary` over `--{ether,coin}base-address`.

**Comment by gavofyork** [comment]
Could of minor alterations would be nice, but good otherwise :)

File: eth/main.cpp (hunk 1 above)
Comment (imifos) [inline eth/main.cpp]: Agree, I did that before my first 0xc0ffee :/

File: eth/main.cpp (hunk 2 above)
Comment (imifos) [inline eth/main.cpp]: Yeah! Totally great (/me hyperbole fan)! That would make it totally clear to users!

**Comment by imifos** [comment]
Corrected the pre-fix typo and removed the coin- to etherbase modification.

Changing the coinbase to for example beneficiary (as discussed), would break compatibility and should be done the correct way, via issue etc.

**Comment by gavofyork** [comment]
rebase needed.

**Pull request merged**